.github/actions/audit-accessibility/main.js

```diff
@@ -48,6 +47,7 @@ const startStorybook = () => {
         const port = 6006;
 
         const storybookServer = http.createServer((request, response) => {
+            // @ts-expect-error - type mismatch in response
```

Comment:
false positive. I've created an issue in js-toolbox to track these problems

https://github.com/Telefonica/js-toolbox/issues/new